.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
```

Comment:
nit: this publishes packages to npm registry

Reply:
Same - Comments from the default template. Do we've any plans on releasing in the github packages too?

Reply:
I think it's better if we publish to github packages as well, that would add one more job to the workflow, do we have any other concerns? Are you planning to do it as part of the same PR? If not we can raise an issue here: https://github.com/dapr/js-sdk/issues